test/cmd/config-changes.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+RESOURCE_DIR="$(dirname "${BASH_SOURCE}")/../resources"
+
+os::test::junit::declare_suite_start "cmd/create"
+
+os::cmd::expect_success 'make build'
+
+os::cmd::expect_success 'oc new-project config'
+
+os::cmd::expect_success 'oc create configmap test-config --from-file=$RESOURCE_DIR/'
+
+os::cmd::expect_success 'oc create -f $RESOURCE_DIR/test-template.yaml'
+
+os::cmd::expect_success 'oc new-app --template=spark -p MASTER_NAME=master -p WORKER_NAME=worker'
+
+os::cmd::try_until_text 'oc logs dc/master' 'DEBUG'
+
```

Comment:
We can add an additional log test here.  The test should check for the following in master/worker logs

Copying from /etc/config to /opt/spark/conf